scopes/component/remove/recover-cmd.ts

```diff
@@ -9,7 +9,7 @@ export type RecoverOptions = {
 
 export class RecoverCmd implements Command {
   name = 'recover <component-name>';
-  description = 'EXPERIMENTAL. recover soft-removed component(s) from the workspace, or a remote scope';
+  description = 'EXPERIMENTAL. recover component(s) soft-removed from the workspace, or a remote scope';
```

Comment:
@benjgil , @itaymendel , @ranm8 , we need to think about the term here.
We changed the `--soft` flag to `--delete`, so not sure we want to call it soft-removed. I started referring to this kind of remove as "marked-remove" but open to suggestions..

Reply:
I like "soft deleted". The point is that the component has been deleted but not as extreme as total annihilation, and `soft delete` still has the word delete like the flag

Thoughts?

Reply:
This actually won't be relevant very soon due to the changes of bit-remove/bit-delete, so never mind. Marking as resolved.